src/Constants.h

```diff
@@ -95,6 +90,7 @@ constexpr std::array<robot::types::motorid_t, 4> WHEEL_IDS = {
 	robot::types::motorid_t::rearLeftWheel,
 	robot::types::motorid_t::rearRightWheel,
 };
+const double STEER_EPSILON = 1000;
```

Comment:
This should be defined as `extern`, in keeping with everything else. Also, can you document what this constant is? (what it is, what the units are, etc)